pages/login/LoginHandler.php

```diff
@@ -249,6 +249,10 @@ public function requestResetPassword($args, $request)
      */
     public function resetPassword($args, $request)
     {
+        if (!$request->isGet()) {
```

Comment:
I would suggest using...
```php
if ($request->getRequestMethod() === 'HEAD') {
    return;
}
```
...and moving the code block just below `$templateMgr = TemplateManager::getManager($request);`.

Benefits:
- Explicitly checks for a `HEAD` response, which is the use case we're trying to capture
- A `HEAD` response should emulate the `GET` response, except without a response body. A redirect doesn't do that.